docs/getting-started/setting-up-funds/running-a-faucet.md

```diff
@@ -42,6 +42,20 @@ After the chain is up and running locally, you can start the faucet by running t
 gnofaucet serve --chain-id dev MyKey
 ```
 
+:::danger Key MyKey not found
```

Comment:
Hey @iuricmp, thanks for pointing this out, you were a bit ahead of me on documenting this :) 

While I think it is super useful to document common errors, I think doing it this way is a little bit cluttering - this would not be scalable if we keep on adding common errors. 

How I would go about documenting this:
1. accurately define the usages of `gnokey` and `gnofaucet`, ie, document the `--home` argument in both (afaik it's missing in the options list), 
2. mention in this file that `gnofaucet` relies on the `gnokey` home dir to look for keys (similar to what you did, just not under ::danger::, but as normal text)
3. further down the line, have a common errors/issues/faq page for docs, which would be an organized place to dump these things.

Down the line, the plan is to have documentation for tools be auto-exported from the `--help` option of the tool, allowing for in-sync tool and docs development.

I suggest you do 1. & 2., and we can discuss 3 :)

Thanks!